adodb.inc.php

```diff
@@ -658,6 +659,29 @@ function Time() {
 		return false;
 	}
 
+	/**
+	 * Parses the hostname to extract the port.
+	 * Overwrites $this->host and $this->port, only if a port is specified.
+	 * The Hostname can be fully or partially qualified,
+	 * ie: "db.mydomain.com:5432" or "ldaps://ldap.mydomain.com:636"
+	 * Any specified scheme such as ldap:// or ldaps:// is maintained.
+	 *
+	 * @return true
```

Comment:
Is there any added value of returning _true_ vs _void_ ? I would personally use the latter (and get rid of the _return_ statement at line 682). Thoughts ?

Reply:
I was originally thinking it could return TRUE if a port was parsed, and FALSE if not. But we aren't checking the return value currently so I never bothered. My habit is always to have a return value no matter what, but I'm fine with no return at all.